client/src/utils/file.js

```diff
@@ -1,15 +1,12 @@
 import siteConfig from '@config/siteConfig.json';
 
 const {
-  publishing: {
-    maxSizeImage = 10000000,
-    maxSizeGif = 50000000,
-    maxSizeVideo = 50000000,
-  }
+  publishing: { maxSizeImage = 10000000, maxSizeGif = 50000000, maxSizeVideo = 50000000 },
```

Comment:
Was this caused by linting?

Reply:
@skhameneh probably. Hey, this was closed without being merged.